pom.xml

```diff
@@ -40,8 +40,8 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.361.x</artifactId>
-        <version>2025.v816d28f1e04f</version>
+        <artifactId>bom-2.387.x</artifactId>
+        <version>2179.v0884e842b_859</version>
```

Comment:
Can we do this in a separate PR?

Or is it asking too much ? 😓

Reply:
Hi @jetersen 
sure: https://github.com/jenkinsci/gitlab-branch-source-plugin/pull/334